pipeline_dp/dp_engine.py

```diff
@@ -49,12 +49,13 @@ def __init__(self, budget_accountant: 'BudgetAccountant',
     def _add_report_stage(self, text):
         self._report_generators[-1].add_stage(text)
 
-    def aggregate(self, col, params: pipeline_dp.AggregateParams,
+    def aggregate(self, col, public_partitions, params: pipeline_dp.AggregateParams,
```

Comment:
`public_partitions` is optional argument. Could you please put them to the end of the list and make default value` = None`.